samples/client/petstore/kotlin-gson/docs/Pet.md

```diff
@@ -5,10 +5,10 @@
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
 **name** | **kotlin.String** |  | 
-**photoUrls** | **kotlin.Array&lt;kotlin.String&gt;** |  | 
```

Comment:
Your change will cause docs to display incorrect type information:

![image](https://user-images.githubusercontent.com/109659/73808296-60effe80-479d-11ea-98de-2f51b1878609.png)

I think this will require triple-braces in the docs markdown files, but I'm not sure how that would work with the cleared escapers.

Why not change the curly braces where you're seeing incorrect generation of HTML entities rather than remove HTML entity processing in general? The HTML escaping exists to avoid potential security risks when users generate code from a compromised server.

I saw your comment in the linked issue saying that the codegen handles html escaping. That's not true exactly: it's the escaper that has been removed in this PR.